### PR TITLE
Add support for multiple MPT3SAS SAS adapters

### PR DESCRIPTION
**Pull request by cmharr**
Currently there are mptsas and mpt2sas checks but those are older generations of cards. This patch uses the exact same logic but replaces '2' with '3' and returns the mpt3sas HBAs.

### CODE REVIEW

**Review by garlick — APPROVED** [review]
Thanks @cmharr.  Has this been tested with the real hardware?   I'll hold off merging until that is confirmed.

**Comment by cmharr** [comment]
@garlick, Yes - the test passed for # of cards, but in our case, we have 2 different types of cards and the logic doesn't allow for separate FW and BIOS versions, so before we merge I'll try to get that new capability added in.
```
[root@goblin5:nodediag.d]# nodediag mpt3sas -v
Checking mpt3sas:                                           [ FAIL ]
1..5
ok 1 - mpt3sas: 2 cards
ok 2 - mpt3sas: host0 fw '17.00.00.00'
ok 3 - mpt3sas: host0 bios '09.33.00.00'
not ok 4 - mpt3sas: host7 fw '07.00.01.00', expected '17.00.00.00'
not ok 5 - mpt3sas: host7 bios '09.13.00.00', expected '09.33.00.00'
```

**Comment by cmharr** [comment]
New PR allows for different versions of SAS cards.
```
[root@goblin5:nodediag.d]# nodediag mpt3sas -v 
Checking mpt3sas:                                           [  OK  ]
1..5
ok 1 - mpt3sas: 2 cards
ok 2 - mpt3sas: host0 fw '17.00.00.00'
ok 3 - mpt3sas: host0 bios '09.33.00.00'
ok 4 - mpt3sas: host7 fw '07.00.01.00'
ok 5 - mpt3sas: host7 bios '09.13.00.00'
```

**Review by garlick — COMMENTED** [review]
OK, great.  When you're ready could you squash those commits so we don't have the incremental changes in the git history?

Also, after squashing, please fix the commit message to conform to 50 chars or less for the summary line, followed by blank line, followed by long description (wrapped at 80 chars)?

**Review by garlick — COMMENTED** [review]
Thanks, ready to merge?

**Comment by cmharr** [comment]
Yes - should be ready to merge.

**Pull request merged**
